modules/effects/src/index.ts

```diff
@@ -1,4 +1,5 @@
 export { createEffect } from './effect_creator';
+export { EffectConfig } from './models';
```

Comment:
This deserves to be exported and documented, IMO.

Reply:
This commit should be a `feat(effects)` instead of `docs(effects)`

Reply:
Fixed 👍